modules/storage-persistent-storage-pvc.adoc

```diff
@@ -113,3 +113,63 @@ spec:
 <1> Path to mount the volume inside the pod.
 <2> Name of the volume to mount. Do not mount to the container root, `/`, or any path that is the same in the host and the container. This can corrupt your host system if the container is sufficiently privileged, such as the host `/dev/pts` files. It is safe to mount the host by using `/host`.
 <3> Name of the PVC, that exists in the same namespace, to use.
+
+[id="pvc-cli-command-usage_{context}"]
+== Viewing PVC usage statistics
+
+You can view usage statistics for persistent volume claims (PVCs).
+
+:FeatureName: PVC usage statistics command
+include::snippets/technology-preview.adoc[leveloffset=+1]
```

Comment:
The epic ([STOR-2040](https://issues.redhat.com/browse/STOR-2040)) describes this feature as "experimental". That's not quite the same as tech preview... do we have an "experimental" snippet? Or would "dev preview" be more appropriate? It's possible for example that the output (API) could still change before this feature goes GA.

Reply:
@sferich888 says tech preview is correct, so nevermind my comment above, you can mark this resolved :)

Reply:
Staying with TP status as per @sferich888